test/e2e/test_e2e_release.sh

```diff
@@ -0,0 +1,96 @@
+#!/bin/bash
+# SPDX-FileCopyrightText: Copyright (c) 2019-2025 Aibolit
+# SPDX-License-Identifier: MIT
+
+set -e
+set -o pipefail
+
+echo "Starting e2e release test..."
```

Comment:
This is great to have this test of building the package, installing it and running a test case! I believe it even deserves a separate PR.

Reply:
@AntonProkopyev Since #825 is already there, can you revert the changes related to e2e testing? The PR description should also be updated.